src/cloudWatchLogs/explorer/logGroupNode.ts

```diff
@@ -0,0 +1,43 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { CloudWatchLogs } from 'aws-sdk'
+import * as os from 'os'
+import { Uri } from 'vscode'
+import { ext } from '../../shared/extensionGlobals'
+import { AWSResourceNode } from '../../shared/treeview/nodes/awsResourceNode'
+import { AWSTreeNodeBase } from '../../shared/treeview/nodes/awsTreeNodeBase'
+
+export const CONTEXT_VALUE_CLOUDWATCH_LOG = 'awsCloudWatchLogNode'
+
+export class LogGroupNode extends AWSTreeNodeBase implements AWSResourceNode {
+    public constructor(
+        public readonly parent: AWSTreeNodeBase,
+        public readonly regionCode: string,
+        public configuration: CloudWatchLogs.LogGroup
+    ) {
+        super('')
+        this.update(configuration)
+        this.iconPath = {
+            dark: Uri.file(ext.iconPaths.dark.cloudWatchLogGroup),
+            light: Uri.file(ext.iconPaths.light.cloudWatchLogGroup),
+        }
+        this.contextValue = CONTEXT_VALUE_CLOUDWATCH_LOG
+    }
+
+    public update(configuration: CloudWatchLogs.LogGroup): void {
+        this.configuration = configuration
+        this.label = this.configuration.logGroupName || ''
```

Comment:
No idea. The type allows it though. Nothing would break if this returned `''`, at least in the tree view (the to-be-added `View Log Streams...` command may though)